testsuite/features/qam/add_custom_repositories/add_ubuntu2004_repositories.feature

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@ubuntu2004_minion
```

Comment:
We might want to register the new tags here `testsuite/features/support/env.rb`
Same for `ubuntu2004_sshminion` tag.

Reply:
oops, yes